packets/hci_packets.pdl

```diff
@@ -2232,9 +2232,7 @@ packet WriteSynchronousFlowControlEnableComplete : CommandComplete (command_op_c
 }
 
 packet SetControllerToHostFlowControl : Command (op_code = SET_CONTROLLER_TO_HOST_FLOW_CONTROL) {
-  acl : 1,
-  synchronous : 1,
-  _reserved_ : 6,
+  enable : Enable,
```

Comment:
The original definition was correct; but if you prefer an enum definition instead, the change would be:

```
enum FlowControlEnable : 8 {
  DISABLED = 0,
  ENABLED_FOR_ACL_PACKETS = 1,
  ENABLED_FOR_SCO_PACKETS = 2,
  ENABLED_FOR_ACL_AND_SCO_PACKETS = 3,
}
```